scripts/_check_docs_utils.py

```diff
@@ -0,0 +1,726 @@
+# coding: utf-8
+
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Utility methods for docstring checking."""
+
+from __future__ import absolute_import
+from __future__ import print_function
+
+import re
+
+import astroid
+
+from pylint.checkers import utils
+
+
+def space_indentation(s):
+    """The number of leading spaces in a string
+
+    :param str s: input string
```

Comment:
Change the doc style to the one we're using here :)

Reply:
Along with the rest of the file

Reply:
Done. Changed all docstrings to match our style. Thanks!